Android/app/src/main/java/com/example/androidgui/ServeDishActivity.java

```diff
@@ -0,0 +1,15 @@
+package com.example.androidgui;
+
+import android.widget.TextView;
+import androidx.appcompat.app.AppCompatActivity;
+import android.os.Bundle;
+
+public class ServeDishActivity extends AppCompatActivity {
+    @Override
+    public void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.staff);
+        TextView header = (TextView) findViewById(R.id.header);
+        header.setText("Serve Dish System");
```

Comment:
Don't use string literal, use Android resources instead

Reply:
Resolved in commit 44254f77c4d79245bf4b70fe7f702b8f1a71dc3f